assets/airports/kaus.json

```diff
@@ -0,0 +1,4024 @@
+                ["edv", 1],
+                ["n/fastGA", 1]
+           ]
+        },
+        {
+            "origin": "",
+            "destination": "KAUS",
+            "category": "arrival",
+            "route": "HFMAN.DXEEE1.KAUS17R",
+            "altitude": [18000, 36000],
+            "speed": 320,
+            "method": "random",
+            "rate": 0.87,
+            "airlines": [
+                ["amx/short", 1],
+                ["ups", 1],
+                ["xa/fastGA", 1]
+           ]
+        },
+        {
+            "origin": "",
+            "destination": "KAUS",
+            "category": "arrival",
+            "route": "HOOLR.DXEEE1.KAUS17R",
+            "altitude": [18000, 36000],
+            "speed": 320,
+            "method": "random",
+            "rate": 0.08,
+            "airlines": [
+                ["swa", 1],
+                ["n/fastGA", 1]
+           ]
+        },
+        {
+            "origin": "",
+            "destination": "KAUS",
+            "category": "arrival",
+            "route": "TPL..CWK",
+            "altitude": 8000,
+            "speed": 180,
+            "method": "random",
+            "rate": 0.75,
+            "airlines": [
+                ["bvn", 3],
+                ["n", 15]
+           ]
+        },
+        {
+            "origin": "",
+            "destination": "KAUS",
+            "category": "arrival",
+            "route": "BWD..CWK",
+            "altitude": 8000,
+            "speed": 180,
+            "method": "random",
+            "rate": 0.67,
+            "airlines": [
+                ["bvn", 1],
+                ["n", 15]
+           ]
+        },
+        {
+            "origin": "",
+            "destination": "KAUS",
+            "category": "arrival",
+            "route": "SJT..CWK",
+            "altitude": 9000,
+            "speed": 180,
+            "method": "random",
+            "rate": 0.79,
+            "airlines": [
+                ["bvn", 4],
+                ["n", 15]
+           ]
+        },
+        {
+            "origin": "",
+            "destination": "KAUS",
+            "category": "arrival",
+            "route": "DBOYS.WLEEE5.KAUS17L",
+            "altitude": [18000, 36000],
+            "speed": 320,
+            "method": "random",
+            "rate": 0.12,
+            "airlines": [
+                ["jbu", 2],
+                ["n/fastGA", 1]
+           ]
+        },
+        {
+            "origin": "",
+            "destination": "KAUS",
+            "category": "arrival",
+            "route": "MNURE.WLEEE5.KAUS17L",
+            "altitude": [18000, 36000],
+            "speed": 320,
+            "method": "random",
+            "rate": 0.75,
+            "airlines": [
+                ["ash", 2],
+                ["n/fastGA", 1],
+                ["swa", 5],
+                ["ual", 7],
+                ["ups", 2],
+                ["xa/fastGA", 1]
+           ]
+        }
+   ],
+    "maps": {
+        "base": [

```

Comment:
972: 
![image](https://user-images.githubusercontent.com/19408472/43342821-fc840072-91db-11e8-8313-6f1d494b7fea.png)
Extended centrelines for 35L/35R?